src/inference/tests/functional/caching_test.cpp

```diff
@@ -2404,8 +2406,8 @@ TEST_P(CachingTest, Load_threads) {
         MkDirGuard guard(cacheDir);
         EXPECT_CALL(*mockPlugin, compile_model(_, _, _)).Times(0);
         EXPECT_CALL(*mockPlugin, compile_model(A<const std::shared_ptr<const ov::Model>&>(), _)).Times(1);
-        EXPECT_CALL(*mockPlugin, import_model(_, _, _)).Times(0);
-        EXPECT_CALL(*mockPlugin, import_model(_, _)).Times(THREADS_COUNT - 1);
+        EXPECT_CALL(*mockPlugin, import_model(A<std::istream&>(), _, _)).Times(0);
+        EXPECT_CALL(*mockPlugin, import_model(A<std::istream&>(), _)).Times(THREADS_COUNT - 1);
```

Comment:
Seem it missing the ov::Any version import_model plugin API mock test?

Reply:
That should be done in scope of enabling mmap in the Mock plugin. Here is the compilation fix only.